src/main/java/net/sf/jabref/model/metadata/MetaData.java

```diff
@@ -147,6 +151,23 @@ public boolean isProtected() {
         return isProtected;
     }
 
+    public Optional<ContentSelectors> getContentSelectors() {
+        return Optional.ofNullable(contentSelectors);
+    }
+
+    public void addContentSelector(ContentSelector contentSelector) {
+        this.contentSelectors.addContentSelector(contentSelector);
+        postChange();
+    }
+
+    public void clearContentSelectors(String fieldName) {
+        contentSelectors.removeSelector(fieldName);
+    }
+
+    public List<String> getContentSelectors(String fieldName) {
```

Comment:
rename to `getContentSelectorValuesForField` or really return the content selector. By the way, the name `ContentSelector` is really strange. It is more a provider than a selector...